workspace_tools/targets.py

```diff
@@ -842,6 +842,23 @@ def __init__(self):
         NRF51822.__init__(self)
         self.extra_labels = ['NORDIC', 'MCU_NRF51822', 'MCU_NORDIC_16K']
         self.macros = ['TARGET_NRF51822']
+		
+class DELTA_DFCM_NNN40(NRF51822):
+    def __init__(self):
+        NRF51822.__init__(self)
+        self.core = "Cortex-M0"
+        self.extra_labels = ['NORDIC', 'MCU_NRF51822', 'MCU_NORDIC_16K']
+        self.macros = ['TARGET_NRF51822']
+        self.supported_toolchains = "GCC_ARM"
```

Comment:
Hi,
It is better if our target were supported ARM or uARM.
But I don't know how to do it.
Just add word string in the toolchains list?
Or I need to do some verify for ARM/uARM?

And I will remove the two json files.
Is it necessary to pull request again when I remove the json files?

Thank you very much
Marco